pkg/proxy/ipvs/proxier.go

```diff
@@ -577,24 +577,40 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	}
 	ipvsModules := utilipvs.GetRequiredIPVSModules(kernelVersion)
 
+	var bmods []string
+
+	// Check if this is a full static Kernel to bypass the next checks
+	_, err = ioutil.ReadFile("/proc/modules")
+	if err != nil {
+		klog.Warningf("Failed to read file /proc/modules with error %v. Assuming this is a fully statically compiled Kernel", err)
+		kernelConfigFile := fmt.Sprintf("/boot/config-%s", kernelVersionStr)
```

Comment:
Are we confident this check here works across all Linux distributions/kernels? I'm okay with the latter half of this PR where we don't re-run `modprobe` if there's already a match, but unsure about this part.

Reply:
About the /proc/modules or about the /boot/config?

About the /boot/config no, I'm far away of being confident but. As I wrote [here](https://github.com/kubernetes/kubernetes/issues/77493#issuecomment-528419879) when I was recompiling the Kernel, I simply didn't copied the file to /boot so in my case this was missing.

On the other hand, this 'if' only exists on statically compiled Kernel (that lacks /proc/modules path) so this case might be pretty particular:

* A very specific distro with a full static kernel (something of IoT maybe?)
* A sysadmin who recompiles the kernel (as our original case/issue) that knows he needs to copy the file (and maybe we should put this into a better logging).